### PR TITLE
Add: 1.11.0-beta2 announcement

### PR DESCRIPTION
**Pull request by TrueBrain**

### CODE REVIEW

File: _posts/2021-02-28-openttd-1-11-0-beta2.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Today we release our second beta of our 1.11 release series!
+
+This second beta is a huge one: we spend the time between last beta and this on massively improving the performance of our video-drivers on all platforms.
+With a huge thanks to michi_cc, we now even have OpenGL support!
+Don't get your hopes up: it still is (and always will be) a 2D game.
+
+To make sure these changes get their fair share of testing, we ask you to help us test them. Especially the following new features:
+* OpenGL support for all platforms.
+* Massive improvements to all video-drivers; so massive, we now have a setting to indicate an upper-limit to fast-forward speeds.
+* Display now runs on 60fps.
```
Comment (LordAro) [inline _posts/2021-02-28-openttd-1-11-0-beta2.md:15]: "or higher, if you alter the refresh-rate setting in the config file" ?

**Comment by TrueBrain** [comment]
Text for socials:

```
Second beta of our 1.11 release series just got released.
60fps, OpenGL support, and much more!
https://www.openttd.org/news/2021/02/28/openttd-1-11-0-beta2.html
```

File: _posts/2021-02-28-openttd-1-11-0-beta2.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+With this new beta, we are slowly wrapping up 1.11, meaning we are closing in on a feature-freeze: a period of time we do not accept new features, but focus on any and all bugs we are aware of.
+This means it is a perfect time for you to report any you have!
+
+Please do help us test, and let us know how you like the improvements we made!
+
+* [Download](https://www.openttd.org/downloads/openttd-releases/testing.html)
+* [Changelog](https://cdn.openttd.org/openttd-releases/1.11.0-beta1/changelog.txt)
+* [Bug tracker](https://github.com/OpenTTD/OpenTTD/issues)
+
+Language without translator:
```
Comment (LordAro) [inline _posts/2021-02-28-openttd-1-11-0-beta2.md:34]:
```suggestion
Languages without translators:
```

File: _posts/2021-02-28-openttd-1-11-0-beta2.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+This second beta is a huge one: we spend the time between last beta and this on massively improving the performance of our video-drivers on all platforms.
+With a huge thanks to michi_cc, we now even have OpenGL support!
+Don't get your hopes up: it still is (and always will be) a 2D game.
+
+To make sure these changes get their fair share of testing, we ask you to help us test them. Especially the following new features:
+* OpenGL support for all platforms.
+* Massive improvements to all video-drivers; so massive, we now have a setting to indicate an upper-limit to fast-forward speeds.
+* Display now runs on 60fps.
+* Generic linux builds.
```
Comment (LordAro) [inline _posts/2021-02-28-openttd-1-11-0-beta2.md:16]:
```suggestion
* Generic Linux builds.
```

File: _posts/2021-02-28-openttd-1-11-0-beta2.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+title: OpenTTD 1.11.0-beta2
+author: TrueBrain
+---
+
+Today we release our second beta of our 1.11 release series!
+
+This second beta is a huge one: we spend the time between last beta and this on massively improving the performance of our video-drivers on all platforms.
```
Comment (LordAro) [inline _posts/2021-02-28-openttd-1-11-0-beta2.md:8]:
```suggestion
This second beta is a huge one: we have spent much of the time between last beta and this on massively improving the performance of our video-drivers on all platforms.
```

File: _posts/2021-02-28-openttd-1-11-0-beta2.md (hunk 1 above)
Comment (TrueBrain) [reply _posts/2021-02-28-openttd-1-11-0-beta2.md:15]: Let's for now keep this a secret :)

File: _posts/2021-02-28-openttd-1-11-0-beta2.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+To make sure these changes get their fair share of testing, we ask you to help us test them. Especially the following new features:
+* OpenGL support for all platforms.
+* Massive improvements to all video-drivers; so massive, we now have a setting to indicate an upper-limit to fast-forward speeds.
+* Display now runs on 60fps.
+* Generic Linux builds.
+
+For a complete list of changes, please see the changelog.
+
+Additionally, we are still searching for translators for 19 languages! 
+Below is the list of language that could really use a translator or two.
```
Comment (LordAro) [inline _posts/2021-02-28-openttd-1-11-0-beta2.md:21]:
```suggestion
Below is the list of languages that could really use a translator or two.
```

File: _posts/2021-02-28-openttd-1-11-0-beta2.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+If you can read/write any of these language, and are willing to help us out make the game look better in that language, please drop by https://translator.openttd.org/ and help us out!
```
Comment (LordAro) [inline _posts/2021-02-28-openttd-1-11-0-beta2.md:22]:
```suggestion
If you can read/write any of these languages, and are willing to help us out make the game look better in that language, please drop by https://translator.openttd.org/ and help us out!
```

**Review by LordAro — APPROVED** [review]

**Pull request merged**